src/lib/queries/getMainCurrency.ts

```diff
@@ -1,32 +1,20 @@
 import { Account, Commodity } from '@/book/entities';
 
 /**
- * In the current database schema there is no setting for storing the mainCurrency
- * gnucash stores this setting somewhere but seems it's internal. To workaround this
- * we are providing this query that counts the commodity used for each INCOME/EXPENSE
- * account. The one most common is assigned as mainCurrency.
- *
- * This function assumes users will have their INCOME and EXPENSE usually with the same
- * currency. If for some reason this is not the case, if may be buggy and change
- * mainCurrency
- *
- * If we don't find one, returns null
+ * We consider the main currency to be the currency of the topmost
+ * ASSET account
```

Comment:
Simplifying the main currency loading. We don't want it to change dynamically but just picking what the root asset account has. As future work, we should disable the users from being able to update placeholder accounts commodity (once we have an account update form).